docs/changelog.rst

```diff
@@ -1,6 +1,17 @@
 CHANGELOG
 =========
 
+Raster Vision 0.13
+-------------------
+
+Features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* It is now possible to use images with more than 3 bands for the semantic segmentation task. Raster Vision automatically modifies the PyTorch DeepLab model to accommodate the additional channels. If using a pretrained model, the pretrained weights for the RGB bands are retained. When using more than 3 bands, Raster Vision creates/consumes training/validation/prediction chips in the form of numpy arrays.
```

Comment:
For features with associated PRs we just write the name of the PR and a link to it. See example here: https://github.com/azavea/raster-vision/blob/master/docs/changelog.rst#raster-vision-011

There were more verbose notes for the 0.12 release which was an anomaly because it was a big refactoring that was spread across many PRs.

Reply:
Got it